[16.0][ADD] shopfloor_reception_putinpack_restriction: allow to restrict the use of packages in reception (#1163)
OCA/wms: shopfloor_reception_putinpack_restriction/readme/CONTEXT.md

> **BUSINESS NEED:**

Reviewer: Why not including this in DESCRIPTION.md?

Author: I am simply following the template of the scaffolding tool that I use: https://github.com/acsone/bobtemplates.odoo

This module allows administrators to simplify the shopfloor user interface by hiding package-related actions during the reception scenario.

> It is specifically designed for warehouses that do not utilize packages during their receptions, preventing warehouse operators from accidentally creating unwanted packages or cluttering the UI with unused buttons.

Reviewer: ```suggestion
It is specifically designed for warehouses that want to control if goods are put in a pack or cannot be put in a pack, here at the end of the reception. This prevents warehouse operators from freely choosing by hiding forbidden buttons.
```

**APPROACH:**

> You can configure on the picking type level if you want to use packages or not.

Reviewer: What about moving this to CONFIGURATION.md ?

```suggestion
You can configure on the picking type if you want to only use packages or forbid them.
```

**USEFUL INFORMATION:**

It can also inform on related modules:

> - modules it depends on: `stock_picking_putinpack_restriction`

Reviewer: Do we need this?
